Adds Email and EmailStrict categories (2022-02-24)

patjack opened the pull request:
Please See: https://github.com/disconnectme/disconnect-tracking-protection/commit/3cc4fca2946654d6a0987f3a1316d80069cce521
https://github.com/disconnectme/disconnect-tracking-protection/commit/e8e755f151af42e7a8884c52f4db4879579f5b30
https://github.com/disconnectme/disconnect-tracking-protection/commit/44fceba8494af171ef0f7fefa002d27e2de5a78a

Please let us know if you have any questions, @say-yawn and team. Thanks!

say-yawn commented:
@patjack, it seems there are more changes than just email trackers being added and wanted to get some clarification on these changes:
```
Advertising entities do not match – diff is {'Salesforce', 'Salesforce.com', 'Acoustic', 'Krux', 'AK', 'Silverpop', 'Pardot'}
URIs do not match for entity Neustar – diff is {'agkn.com', 'aggregateknowledge.com'}
Content entities do not match – diff is {'Salesforce', 'Salesforce.com'}
Analytics entities do not match – diff is {'ZoomInfo'}
FingerprintingGeneral entities do not match – diff is {'Salesforce', 'Krux'}
```
Are these because they are being moved to the email trackers?

patjack commented:
> 

Hi @say-yawn, thanks for reviewing and sharing your diff. All these changes (with the exception of ZoomInfo described in more detail below), are related to the new Email trackers. In the process of researching the email trackers we added, we did rename or combine some of the current entities on the list. Hopefully the details below add clarity. Let me know if you have more questions or thoughts.

> Advertising entities do not match – diff is {'Salesforce', 'Salesforce.com', 'Acoustic', 'Krux', 'AK', 'Silverpop', 'Pardot'}

While adding these email trackers, we changed Salesforce.com > Salesforce; Silverpop -> Acoustic; AK -> Neustar, Moved Krux and Pardot under the Salesforce entity.
>URIs do not match for entity Neustar – diff is {'agkn.com', 'aggregateknowledge.com'}

This was apart of the change to move the 'AK' entity to the 'Neustar' entity
>Content entities do not match – diff is {'Salesforce', 'Salesforce.com'}

This is because the "Salesforce.com" entity was renamed to "Salesforce"
>Analytics entities do not match – diff is {'ZoomInfo'}

This was [committed](https://github.com/disconnectme/disconnect-tracking-protection/commit/8f8a3444c39500a2bd2b8e7b7a5e650f928b95c7) to the Disconnect repo. It looks like we never pushed this commit as a PR to shavar. Would it help if I created a new PR with just this change or is it OK to keep it here?

>FingerprintingGeneral entities do not match – diff is {'Salesforce', 'Krux'}

This is due to combining the Krux entity with Salesforce.

say-yawn reviewed (approved):
@patjack Thanks for the explanation! LGTM.
